bundles/org.openhab.binding.keba/src/main/resources/ESH-INF/thing/kecontact.xml

```diff
@@ -245,4 +246,10 @@
 		<description>Session ID of the last charging session</description>
 		<state readOnly="true"></state>
 	</channel-type>
+	<channel-type id="setenergy">
```

Comment:
Maybe a better name for this channel would be something like `sessionlimit` or `sessioncharginglimit`? Those are properties of the Thing whereas `setenergy` seems to be an operation to modify the property of the Thing.

Reply:
The name was choosen from the documentation of the manufacturer. But sure the name setenergy could be missleading as thies is really changing the behaviour ( the charging will be interrupted once the limit is reached). I will change it to `"setenergylimit"` , to indicate that this sets a limit and to stay close to the documentation. Charginglimit could be also missleading as the charging is accounted in kW but this is a energy limit for kWh. Does this sounds right?

Reply:
Sounds right to me.